health/QUICKSTART.md

```diff
@@ -87,7 +87,10 @@ lookup: average -1m percentage of used
 
 Let's look into each of the lines to see how they create a working health entity.
 
--   `alarm`: The name for your new entity. The name can be anything, but the only symbols allowed are `.` and `_`.
+-   `alarm`: The name for your new entity. The name needs to follow the requirements:
```

Comment:
```suggestion
-   `alarm`: The name for your new entity. The name needs to follow these requirements:
```

Reply:
> Just wanted to make the two lists the same!
> 
> Can you add 2 more spaces after the `-` in the sub-items? Otherwise we'll add some Markdown linter errors. wink
> 
> ```
> -   first level
>      -   second level
>      -   second level
>      -   second level
> ```

Everything was changed as you request, thank you very much! :)